packages/feedback/src/Feedback.js

```diff
@@ -36,20 +39,26 @@ const Feedback = ({
         <FeedbackModal
           onFeedbackSent={onFeedbackSent}
           prompt={prompt}
-          isOpen={isOpen}
+          isOpen={feedbackIsOpen}
           zIndex={zIndex}
-          toggle={() => toggle()}
+          toggle={() => feedbackToggle()}
           name={appName}
           analytics={analytics}
+          showSupport={showSupport}
```

Comment:
I think we should also list the `supportZIndex` prop here to be clear even though `FeedbackModal` will destructure it from `{...formProps}`. The rest LGTM to me though!

Reply:
See 0450fa4e612688211cc7c61ce0b0c8591dbb42ce